test/Microsoft.Health.Fhir.Shared.Tests.Integration/Persistence/SqlServerSchemaUpgradeTests.cs

```diff
@@ -75,42 +75,34 @@ public async Task GivenTwoSchemaInitializationMethods_WhenCreatingTwoDatabases_B
             }
         }
 
-        [Theory]
-        [MemberData(nameof(GetSchemaVersions))]
-        public async Task GivenASchemaVersion_WhenApplyingDiffTwice_ShouldSucceed(int schemaVersion)
+        [Fact]
+        public void GivenASchemaVersion_WhenApplyingDiffTwice_ShouldSucceed()
         {
-            var snapshotDatabaseName = $"SNAPSHOT_{DateTimeOffset.UtcNow.ToUnixTimeSeconds()}_{BigInteger.Abs(new BigInteger(Guid.NewGuid().ToByteArray()))}";
-
-            SqlServerFhirStorageTestHelper testHelper = null;
-            SchemaUpgradeRunner upgradeRunner;
-
-            try
+            Parallel.ForEach(Enum.GetValues(typeof(SchemaVersion)).OfType<int>().ToList(), async version =>
             {
-                (testHelper, upgradeRunner) = await SetupTestHelperAndCreateDatabase(
-                    snapshotDatabaseName,
-                    schemaVersion - 1,
-                    forceIncrementalSchemaUpgrade: false);
+                if (version >= 7)
+                {
+                    var snapshotDatabaseName = $"SNAPSHOT_{DateTimeOffset.UtcNow.ToUnixTimeSeconds()}_{BigInteger.Abs(new BigInteger(Guid.NewGuid().ToByteArray()))}";
 
-                await upgradeRunner.ApplySchemaAsync(schemaVersion, applyFullSchemaSnapshot: false, CancellationToken.None);
-                await upgradeRunner.ApplySchemaAsync(schemaVersion, applyFullSchemaSnapshot: false, CancellationToken.None);
-            }
-            finally
-            {
-                await testHelper.DeleteDatabase(snapshotDatabaseName);
-            }
-        }
+                    SqlServerFhirStorageTestHelper testHelper = null;
+                    SchemaUpgradeRunner upgradeRunner;
 
-        public static IEnumerable<object[]> GetSchemaVersions()
-        {
-            foreach (object item in Enum.GetValues(typeof(SchemaVersion)))
-            {
-                // The schema upgrade scripts starting from v7 were made idempotent.
```

Comment:
Could you add this comment back to the if statement on line 83? I think it is important to remember why version 7 is important.

Reply:
Yes, I added the comment